app/User.php

```diff
@@ -213,6 +213,18 @@ public function scopeName($query, $value)
         return $query->orWhereRaw("CONCAT(first_name,' ',last_name) ILIKE '%" . $value . "%'");
     }
 
+    /**
+     * Scope for email search
+     *
+     * @param $query
+     * @param $value
+     * @return mixed
+     */
+    public function scopeSearch($query, $value)
+    {
+        return $query->orWhereRaw("email ILIKE '%" . $value . "%'")->orWhereRaw("CONCAT(first_name,' ',last_name) ILIKE '%" . $value . "%'");
```

Comment:
Need space after comma(,) in argument list.
orWhereRaw("CONCAT(first_name,' ',last_name) ILIKE '%" . $value . "%'");

Reply:
Updated.